src/main/java/edu/kis/powp/jobs2d/command/visitor/TestVisitor.java

```diff
@@ -0,0 +1,46 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.Iterator;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class TestVisitor implements IVisitor {
+	private final Logger logger;
+	private int operateToCommandCount = 0;
+	private int setPositionCommandCount = 0;
+	private int compundCommandCount = 0;
```

Comment:
Provide getters, so visitor can be reused. You can also add:
getCommandsTotalCount
getAtomicCommandsCount